apps/site/assets/css/_header.scss

```diff
@@ -358,6 +358,18 @@ nav.m-menu--mobile {
   }
 }
 
+.m-menu__toggle {
+  width: 5em;
```

Comment:
Looking into this issue a bit more it seems like the issue is tied to whether or not the scrollbar is showing or not and the container getting wider to fill the extra space when the search overlay is show and subsequently hides the scrollbar.

Because of this I think this solution may only resolve the issue for some browser types due to the differences in scroll bar widths etc across different platforms. 

A possible solution to resolve the issue across all platforms is to add a `width: 100vw` to the `<body>` element to always have the body expanded to the entire viewport and not subtract the scrollbar width. But this could potentially cause undesirable results due to seemingly mismatched margins/padding on either side depending on the browser and may be a question for design.

Reply:
Ah I see what you're saying now, I was testing using responsive mode and my scrollbars were the kind that hovers over the content instead of being next to it. There's definitely a much more noticeable shift when the scrollbar pops in and out with the default scrollbars (though this is still solving the very small amount of shifting I see in responsive design mode).

I'll noodle on this a bit, thanks for the call out 👍

Reply:
I've logged a separate ticket here for the desktop fixes: https://app.asana.com/0/555089885850811/1201920171846633/f

I think we can safely merge this in to fix the minor issue on mobile while thinking about how to address the bigger issue on desktop